### PR TITLE
Move core UI

### PR DESCRIPTION
**Pull request by qbalukom**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,40 +4,133 @@
...
     "lint-fix": "prettier --config ./.prettierrc --ignore-path ./.prettierignore --write '**/*.{ts,tsx,js,jsx,json,html,css,yaml,md}'",
-    "lint-staged": "lint-staged --concurrent=false"
+    "lint-staged": "lint-staged --concurrent=false",
+    "test": "react-app-rewired test",
+    "eject": "react-scripts eject",
+    "build": "npm run copy-themes; INLINE_RUNTIME_CHUNK=false GENERATE_SOURCEMAP=false react-app-rewired build",
+    "build:docker": "npm run copy-themes; INLINE_RUNTIME_CHUNK=false GENERATE_SOURCEMAP=false react-app-rewired build --env IS_DOCKER=true",
+    "copy-themes": "mkdir -p public/themes/@sap-theming; cp node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/css_variables.css public/themes/@sap-theming/default.css; for theme in hcb dark hcw light_dark ; do cp node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3_$theme/css_variables.css public/themes/@sap-theming/$theme.css ; done"
+  },
+  "eslintConfig": {
```
Comment (mrCherry97) [inline package.json:20]: Please add `    "ignorePatterns": "tests/*",`

**Review by mrCherry97 — CHANGES_REQUESTED** [review]
Please remove `core` folder too

File: Dockerfile.local [hunk 2]
```diff
@@ -33,14 +33,14 @@ RUN apk --no-cache upgrade && \
   apk add --update nodejs npm yq
 WORKDIR /app
 
-COPY --from=builder /app/core-ui/build /app/core-ui
```
Comment (mrCherry97) [inline Dockerfile.local:36]: As we have a talk, please open new task as follow-up to change all `core-ui` from images etc

**Review by mrCherry97 — DISMISSED** [review]

**Review by mrCherry97 — APPROVED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**